wire-compiler/src/test/java/com/squareup/wire/WireCompilerErrorTest.java

```diff
@@ -143,14 +141,15 @@ public ProtoFile parse(String filename) throws IOException {
           + "}\n");
       fail();
     } catch (IllegalStateException e) {
-      assertEquals("Duplicate enum constant QUIX in scope com.squareup.protos.test.Foo", e.getMessage());
+      assertThat(e)
+          .hasMessage("Duplicate enum constant QUIX in scope com.squareup.protos.test.Foo");
     }
   }
 
   @Test public void testNoPackageNameIsLegal() {
     Map<String, String> output = compile("message Simple { optional int32 f = 1; }");
-    assertTrue(output.containsKey(".Simple"));
+    assertThat(output).containsKey(".Simple");
     // Output should not have a 'package' declaration.
-    assertFalse(output.get(".Simple").contains("package"));
+    assertThat(output.get(".Simple").contains("package")).isFalse();
```

Comment:
Future refactor: replace both with `assertThat(output.get(".Simple")).doesNotContain("package")` (the `null` check comes free).